dashboard/src/pages/settings.tsx

```diff
@@ -0,0 +1,230 @@
+import { client } from '/@/libs/api'
+import { DeleteUserKeyRequest } from '/@/api/neoshowcase/protobuf/gateway_pb'
+import { styled } from '@macaron-css/solid'
+import { vars } from '/@/theme'
+import { Container } from '/@/libs/layout'
+import { Button } from '/@/components/Button'
+import { InputBar, InputLabel } from '/@/components/Input'
+import toast from 'solid-toast'
+import { ConnectError } from '@bufbuild/connect'
+import { style } from '@macaron-css/core'
+
+// copy from /pages/apps AppsTitle component
+const PageTitle = styled('div', {
+  base: {
+    marginTop: '48px',
+    fontSize: '32px',
+    fontWeight: 'bold',
+    color: vars.text.black1,
+  },
+})
+
+// copy from /pages/apps
+// and delete unnecessary styles
+const ContentContainer = styled('div', {
+  base: {
+    marginTop: '24px',
+  },
+})
+
+const SidebarTitle = styled('div', {
+  base: {
+    fontSize: '24px',
+    fontWeight: 500,
+    color: vars.text.black1,
+  },
+})
+
+const MainContentContainer = styled('div', {
+  base: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '20px',
+  },
+})
+
+const UserKeysContainer = styled('div', {
+  base: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '16px',
+
+    background: vars.bg.white3,
+    border: `1px solid ${vars.bg.white4}`,
+    borderRadius: '4px',
+    padding: '8px 12px',
+  },
+})
+
+const PublicKey = styled('div', {
+  base: {
+    background: vars.bg.white1,
+    border: `1px solid ${vars.bg.white5}`,
+    borderRadius: '4px',
+    padding: '8px 12px',
+  },
+})
+
+const PublicKeyContainer = styled('div', {
+  base: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '6px',
+
+    background: vars.bg.white2,
+    border: `1px solid ${vars.bg.white5}`,
+    borderRadius: '4px',
+    padding: '8px 12px',
+  },
+})
+
+const CreateKeyContainer = styled('div', {
+  base: {
+    background: vars.bg.white3,
+    border: `1px solid ${vars.bg.white4}`,
+    borderRadius: '4px',
+    padding: '8px 12px',
+  },
+})
+
+const CreatingKeyContainerClass = style({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: '16px',
+
+  background: vars.bg.white2,
+  border: `1px solid ${vars.bg.white5}`,
+  borderRadius: '4px',
+  padding: '8px 12px',
+})
+styled('form', {
+  base: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '16px',
+
+    background: vars.bg.white2,
+    border: `1px solid ${vars.bg.white5}`,
+    borderRadius: '4px',
+    padding: '8px 12px',
+  },
+})
+export const FormButton = styled('div', {
+  base: {
+    marginLeft: '4px',
+  },
+})
+
+export default () => {
+  const [userKeys, { refetch: refetchApp }] = createResource(() => client.getUserKeys({}))
+
+  const [createKeyToggle, setCreateKeyToggle] = createSignal(false)
+
+  const deleteKeyRequest = (keyID: string) => {
+    try {
+      const a = new DeleteUserKeyRequest()
+      a.keyId = keyID
+      client.deleteUserKey(a)
```

Comment:
awaitしないと間違ったtoastが出ます
```suggestion
  const deleteKeyRequest = async (keyID: string) => {
    try {
      const a = new DeleteUserKeyRequest()
      a.keyId = keyID
      await client.deleteUserKey(a)
```